src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -1523,6 +1524,7 @@ internal bool ExceptionRecorded
         /// <see cref="System.Threading.Tasks.TaskStatus.Faulted">TaskStatus.Faulted</see>, and its
         /// <see cref="Exception"/> property will be non-null.
         /// </remarks>
+        [MemberNotNullWhen(true, nameof(Exception))]
```

Comment:
For this to impact external consumers, you'll also need to update the ref assembly.

Reply:
@stephentoub Are there are instructions I can follow?

Reply:
Just make the same change (except using fully-qualified names) to the corresponding member in https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/ref/System.Runtime.cs

Reply:
@stephentoub Updated.